person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
-  [OPTIONAL] For good code organization, it will be nice to move this to the top of your class.

Reply:
Thank you for the review and the feedback @Mihndim2020 I'll implement all suggestions.